### PR TITLE
Add docs for cors-proxy service.

### PR DESCRIPTION
**Pull request by walac**

### CODE REVIEW

File: services/cors-proxy/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+---
+
+## Cors-Proxy
+
+[cors-proxy](https://github.com/taskcluster/cors-proxy) is a simple service
+to enable taskcluster frontend services to make access remote endpoints that
+disallow cross origin requests.
+
+Any website under `taskcluster.net` domain can make remote requests through
+cors-proxy.
```
Comment (djmitche) [inline services/cors-proxy/index.md]: This isn't quite true anymore :)

**Comment by djmitche** [comment]
Looks good other than rewording that bit.  You could just say "a whitelist", and link to the code, rather than trying to keep the list in docs and the code matching.

**Comment by walac** [comment]
Done.

**Comment by djmitche** [comment]
:+1:

**Pull request merged**